README.md

```diff
@@ -347,11 +357,21 @@ sliced "packet".
 
 Fields:
 
- - `enabled`: true/false
  - `average_size`: size in bytes of an average packet
  - `size_variation`: variation in bytes of an average packet (should be smaller than average_size)
  - `delay`: time in microseconds to delay each packet by
 
+### Writing custom toxics
+
+Writing your own toxics for Toxiproxy can be done by copying [toxiproxy.go](https://github.com/Shopify/toxiproxy/blob/master/cmd/toxiproxy.go)
+into a new project and registering your toxic with the server. This will allow you to build your own Toxiproxy
+binary without having to make a full fork of the project. If you think your toxics will be useful to others,
+contribute them back with a Pull Request.
```

Comment:
Do we really think people are going to import the project and build their own servers though?